skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/plugin/interceptor/enhance/ClassEnhancePluginDefine.java

```diff
@@ -70,8 +69,15 @@
             for (ConstructorInterceptPoint constructorInterceptPoint : constructorInterceptPoints) {
                 newClassBuilder = newClassBuilder.constructor(constructorInterceptPoint.getConstructorMatcher())
                         .intercept(SuperMethodCall.INSTANCE.andThen(
-                        MethodDelegation.to(new ClassConstructorInterceptor(constructorInterceptPoint.getConstructorInterceptor()))
-                                .appendParameterBinder(FieldProxy.Binder.install(FieldGetter.class, FieldSetter.class))));
+                                MethodDelegation.to(new ClassConstructorInterceptor(constructorInterceptPoint.getConstructorInterceptor()))
+                                        .appendParameterBinder(FieldProxy.Binder.install(FieldGetter.class, FieldSetter.class))));
+            }
+        } else {
+            for (ConstructorInterceptPoint constructorInterceptPoint : constructorInterceptPoints) {
+                newClassBuilder = newClassBuilder.constructor(ElementMatchers.<MethodDescription>any())
+                        .intercept(SuperMethodCall.INSTANCE.andThen(
+                                MethodDelegation.to(new DefaultClassConstructorInterceptor(constructorInterceptPoint.getConstructorInterceptor()))
```

Comment:
No, you would need to chain them for this, the last one intercepts the next one. One way to concatenate interceptions would be to use Byte Buddy's `Advice` as a visitor where they are grouped.